README.md

```diff
@@ -1,2 +1,36 @@
 # gh-dependabot
-A GH CLI extension for Dependabot
+A GH CLI extension for using Dependabot
```

Comment:
How broad of a vision behind this extension are you imagining?  Any thoughts how it complements [other dependabot extensions](https://github.com/search?q=topic%3Agh-extension%20dependabot&type=repositories)?

Reply:
Yea...I figured this could be a one stop shop for all things dependabot. The other extensions only deal with the PRs that dependabot opens up so it doesn't touch anything else about dependabot. I figured this extension could do what those extensions do but also be more extensible to do more things surrounding dependabot. Things like enabling/disabling it, configuring the package management configuration and anything else.

As I continue to learn more about dependabot and how to use it and understand the needs around automating dependabot, I want to make this extension be able to tackle that.